docs/production/authentication-methods.md

```diff
@@ -1,14 +1,34 @@
 # Authentication methods
 
-Zulip supports a wide variety of authentication methods. Some of them
-require configuration to set up.
+Zulip supports a wide variety of authentication methods:
+
+- [Email and password](#email-and-password), which is enabled by default.
+- [Social authentication](#social-authentication) with Google, GitHub,
+  and GitLab, which is easy to set up with a few lines of configuration.
+  Authentication with Apple additionally requires registering with Apple.
+- [Microsoft Entra ID](#microsoft-entra-id) (AzureAD), which is similarly easy to
+  configure.
+- [LDAP (including Active Directory)](#ldap-including-active-directory). Zulip
+  supports retrieving information about users via LDAP, and optionally using LDAP
+  as an authentication mechanism.
+- [SAML](#saml), which is used by Okta, OneLogin, Entra ID (AzureAD), Keycloak,
```

Comment:
"which is supported by" is probably clearer.

Reply:
See updated screenshot in main PR comment.